compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -244,16 +240,13 @@ public SymbolEnter(CompilerContext context) {
         this.types = Types.getInstance(context);
         this.typeParamAnalyzer = TypeParamAnalyzer.getInstance(context);
         this.anonymousModelHelper = BLangAnonymousModelHelper.getInstance(context);
-        this.sourceDirectory = context.get(SourceDirectory.class);
         this.importedPackages = new ArrayList<>();
         this.unknownTypeRefs = new HashSet<>();
         this.missingNodesHelper = BLangMissingNodesHelper.getInstance(context);
         this.packageCache = PackageCache.getInstance(context);
         this.intersectionTypes = new ArrayList<>();
 
         CompilerOptions options = CompilerOptions.getInstance(context);
-        projectAPIInitiatedCompilation = Boolean.parseBoolean(
-                options.get(CompilerOptionName.PROJECT_API_INITIATED_COMPILATION));
```

Comment:
The compiler option name can be removed now. https://github.com/ballerina-platform/ballerina-lang/blob/1ee5bb5fc1655413c6d972d1ee57b67e76e68761/compiler/ballerina-lang/src/main/java/org/ballerinalang/compiler/CompilerOptionName.java#L68-L71

Reply:
Added